src/src/components/obComponents/columnFilters/simpleMultiSelect.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <v-select
+        :key="column.key"
+        v-model="modelValue"
+        :items="column.filterType == filterType.SimpleBoolean ? booleanArray : column.valueAccessor"
+        :label="column.title"
+        multiple
+        persistent-hint
+        item-value="id"
+        item-title="name"
+        :autofocus="true"
+        @update:model-value="addFilter()"
+    >
+        <template v-slot:prepend-item>
+            <v-list-item title="Select All" @click="toggleList()">
+                <template v-slot:prepend>
+                    <v-checkbox-btn
+                        :color="checkSomeItems(column.key) ? 'indigo-darken-4' : undefined"
+                        :indeterminate="
+                            checkSomeItems(column.key) &&
+                            !checkAllItems(
+                                column.key,
+                                column.filterType == filterType.SimpleBoolean ? booleanArray : column.valueAccessor,
+                            )
+                        "
+                        :model-value="
+                            checkAllItems(
+                                column.key,
+                                column.filterType == filterType.SimpleBoolean ? booleanArray : column.valueAccessor,
+                            )
+                        "
+                    />
+                </template>
+            </v-list-item>
+        </template>
+    </v-select>
+</template>
+
+<script setup lang="ts">
+    import { toRef, ref } from "vue";
+    import { Column, FilterType } from "@js/grids/vuetify/types";
+
+    const props = defineProps({
+        column: {
+            type: Column,
+            require: true,
+        },
+    });
+    const column = toRef(props, "column");
+    const filterType = FilterType;
```

Comment:
This seems unnecessary

Reply:
We are using filterType above for  column.filterType == filterType.SimpleBoolean ? booleanArray : column.valueAccessor,

Reply:
Yes, but surely you can just change that to `column.filterType == FilterType.SimpleBoolean ? booleanArray : column.valueAccessor,`

Reply:
ok as earlier in vue2 , we are doing that to access enum, so I did , will remove